app/jobs/prepare_person_escort_record_notifications_job.rb

```diff
@@ -3,9 +3,29 @@
 class PreparePersonEscortRecordNotificationsJob < ApplicationJob
   include QueueDeterminer
 
-  def perform(topic_id:, action_name:, **_)
-    PersonEscortRecord.find(topic_id).profile.moves.find_each do |move|
-      PrepareMoveNotificationsJob.perform_now(topic_id: move.id, action_name: action_name, queue_as: move_queue_priority(move))
+  def perform(topic_id:, **_)
+    person_escort_record = PersonEscortRecord.find(topic_id)
+    return unless (move = person_escort_record.move)
+
+    [move.supplier || move.suppliers].flatten.each do |supplier|
```

Comment:
There is a bit of  duplication here, is there something we can do to choose which notification we want and passing it in to `PrepareMoveNotificationsJob` instead?

Reply:
Update: I just confirmed we will need to send both webhooks and emails unfortunately for this notification

Reply:
👍